cmd/kogito/command/flag/monitoring_flag.go

```diff
@@ -15,7 +15,7 @@
 package flag
 
 import (
-	"github.com/kiegroup/kogito-cloud-operator/api/v1beta1"
+	"github.com/kiegroup/kogito-cloud-operator/api"
```

Comment:
shuoldn't we name it as v1 instead use just api?

Reply:
we will introduced `v1` in product-operator. Currently `api` pkg have interface and `v1beta1` contains its implementation.